richreview/public/webapps/richreview/r2/model/doc_model.js

```diff
@@ -1758,7 +1759,12 @@
         canvas_ctx.lineJoin = 'round';
         canvas_ctx.stroke();
     };
-
+    r2.Ink.prototype.smoothing = function(canvas_ctx){
+        var i, segment;
+        for(i = 0; segment = this.segments[i]; ++i) {
+            segment.smoothing();
+        }
+    };
```

Comment:
Cool. Can you re-commit with revised changes?

On Sun, Nov 22, 2015 at 9:36 AM, Yuan Huang <notifications@github.com>
wrote:

> In richreview/public/webapps/richreview/r2/model/doc_model.js
> <https://github.com/DongwookYoon/RichReviewXBlock/pull/16#discussion_r45559607>
> :
>
> > @@ -1758,7 +1759,12 @@
> >          canvas_ctx.lineJoin = 'round';
> >          canvas_ctx.stroke();
> >      };
> > -
> > +    r2.Ink.prototype.smoothing = function(canvas_ctx){
> > +        var i, segment;
> > +        for(i = 0; segment = this.segments[i]; ++i) {
> > +            segment.smoothing();
> > +        }
> > +    };
>
> Why just the last segment? I think all the segments should be smoothed
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/DongwookYoon/RichReviewXBlock/pull/16/files#r45559607>
> .
>



-- 
Best,
Dongwook


Dongwook Yoon
PhD Student
Information Science Department
Cornell University
T: +1-408-444-2877
E: dy252@cornell.edu